README.md

```diff
@@ -48,6 +49,40 @@ current editor using the keymap `ctrl-shift-m`.
 Should you have any problems while installing or using MPP please open up a
 [new issue](https://github.com/Galadirith/markdown-preview-plus/issues/new).
 
+- **I want to use pandoc but it doesn't work**  
```

Comment:
Could you remove L52 and L53. They don't really have a place any more as I removed the troubleshooting section however I don't think that really matters as L43 can point them to the Pandoc section.